src/prompts/password.rs

```diff
@@ -145,6 +160,18 @@ impl<'a> Password<'a> {
         self
     }
 
+    /// Sets the flag to enable verification.
+    pub fn with_verification_enabled(mut self) -> Self {
+        self.enable_verification = true;
+        self
+    }
+
+    /// Sets the verification message of the prompt.
+    pub fn with_verification_message(mut self, message: &'a str) -> Self {
+        self.verification_message = message;
+        self
+    }
```

Comment:
I think the suggestion below of having `"Confirmation:"` being the default confirmation message was quite good, but this is also fine for me. Although, this will be slightly less performant than the other suggestion, but I feel like convenience triumphs in this case :)

Reply:
This was mostly based on the 1 line approach, I too prefer the `Confirmation` option :)